src/ui/server/controllers/insurance/business/company-details/index.test.ts

```diff
@@ -1,16 +1,28 @@
 import { Request, Response } from '../../../../../types';
 import { mockReq, mockRes, mockCompany } from '../../../../test-mocks';
-import { get, postCompaniesHouseSearch } from '.';
+import { get, postCompaniesHouseSearch, post } from '.';
 import { FIELD_IDS, ROUTES, TEMPLATES } from '../../../../constants';
 import corePageVariables from '../../../../helpers/page-variables/core/insurance';
 import { PAGES, ERROR_MESSAGES } from '../../../../content-strings';
 import generateValidationErrors from '../../../../helpers/validation';
 import api from '../../../../api';
 import { companyHouseSummaryList } from '../../../../helpers/summary-lists/company-house-summary-list';
 
-const { COMPANY_HOUSE } = FIELD_IDS.INSURANCE.EXPORTER_BUSINESS;
+const { EXPORTER_BUSINESS } = FIELD_IDS.INSURANCE;
+const { COMPANY_HOUSE, YOUR_COMPANY } = EXPORTER_BUSINESS;
+const { TRADING_NAME } = YOUR_COMPANY;
```

Comment:
Could we do destructuring for each thing inside a const? We're doing this in other places, think it's easier to read?

Reply:
done